Don't override MacPorts CFLAGS and CXXFLAGS. Remove optimization flags
because MacPorts will set them.
--- configure.in.orig	2014-11-20 15:19:10.000000000 -0600
+++ configure.in	2018-12-07 09:58:12.000000000 -0600
@@ -11,9 +11,9 @@
 dnl# add warnings and optimization to compiler switches:
 if test -n "$GXX"; then
     # GNU C compiler
-    CFLAGS="-Wall -O2"
+    CFLAGS+=" -Wall"
     CFLAGS_D="-Wall -g -DDEBUG"
-    CXXFLAGS="-Wall -O2"
+    CXXFLAGS+=" -Wall"
     CXXFLAGS_D="-Wall -g -DDEBUG"
 else
 if test "`(uname) 2>/dev/null`" = IRIX; then
@@ -37,9 +37,7 @@
   fi
 else
     # generic C compiler:
-    CFLAGS="-O"
     CFLAGS_D="-g -DDEBUG"
-    CXXFLAGS="-O"
     CXXFLAGS_D="-g -DDEBUG"
 fi
 fi
